src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -165,6 +164,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
         private void ConfigHttpClients(IServiceCollection services)
         {
+            var acceptHeaderName = "Accept";
```

Comment:
There is inconsistency where other headers are not using variables. Also variables should be declared close to its usage there by limiting its scope. 
I suggest that you get rid of these variables as they are not being used anywhere else.

Reply:
Done